gpioioctl/gpio.go

```diff
@@ -587,24 +622,22 @@ func (d *driverGPIO) Init() (bool, error) {
 var drvGPIO driverGPIO
 
 func init() {
-	if runtime.GOOS == "linux" {
```

Comment:
The driver should still not register on non-linux OSes?

For the bcm283x test to pass, you can force this driver to load manually. I would prefer that.

Reply:
The removal causes the driver.Init() to get called.

driver.Init() is now conditional because it looks at /dev/gpiochip which will not be available on those OSes.

Also, note that gpio_other.go creates a mock.